chain/client/src/info.rs

```diff
@@ -635,22 +630,11 @@ pub fn display_sync_status(
             )
         }
         SyncStatus::StateSync(StateSyncStatus { sync_hash, sync_status: shard_statuses }) => {
-            let mut res = format!("State {:?}", sync_hash);
+            let mut res = format!("State {sync_hash:?}");
             let mut shard_statuses: Vec<_> = shard_statuses.iter().collect();
             shard_statuses.sort_by_key(|(shard_id, _)| *shard_id);
             for (shard_id, shard_status) in shard_statuses {
-                write!(res, "[{}: {}]", shard_id, shard_status.status.to_string(),).unwrap();
-            }
-            if matches!(state_sync_config, SyncConfig::Peers) {
-                // TODO #8719
-                tracing::warn!(
```

Comment:
instead of removing this message, should we replace with a message saying "node is syncing its state! If this failed, please do the following xxx"?